pandas/tests/resample/test_period_index.py

```diff
@@ -811,39 +812,80 @@ def test_resample_with_only_nat(self):
         tm.assert_frame_equal(result, expected)
 
     @pytest.mark.parametrize(
-        "start,end,start_freq,end_freq,base",
+        "start,end,start_freq,end_freq,base,offset",
         [
```

Comment:
remove base from all tests (except for a small number which by-definition are showing FutureWarning)

Reply:
I understand why you said that, but those tests are quite useful IMO. It avoids to break the base argument even if it is now deprecated. For instance to handle the conversion from `base` to `offset` in my last commit, those tests helped me.

I can delete those tests if you want to, but I was thinking: maybe I should just move them into a specific file like `resample/test_deprecated.py`? This file could be easily removed afterwards. :man_shrugging:

Reply:
I just did that in the last commit, I moved all deprecated tests into `test_deprecated.py`